tests/widget/modal/widget-modal.webauthn-login.test.js

```diff
@@ -0,0 +1,40 @@
+import { expect, test } from '@playwright/test';
+import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
+
+test('Modal widget with webauthn login', async ({ browser, page }) => {
+  const { clickButton, navigate } = asyncEvents(page);
+
+  let authenticator;
+  if (browser.browserType().name() === 'chromium') {
+    const cdpSession = await page.context().newCDPSession(page);
+
+    await cdpSession.send('WebAuthn.enable');
+    authenticator = await cdpSession.send('WebAuthn.addVirtualAuthenticator', {
+      options: {
+        protocol: 'ctap2',
+        transport: 'internal',
+        hasUserVerification: true,
+        isUserVerified: true,
+        hasResidentKey: true,
+      },
+    });
+
+    await navigate('widget/modal?journey=TEST_WebAuthn-Registration');
+
+    await clickButton('Open Login Modal', '/authenticate');
+
+    await page.getByLabel('Username').click();
+    await page.getByLabel('Username').fill('demouser');
+    await page.getByLabel('Username').press('Tab');
+
+    await page.getByLabel('Password').fill('j56eKtae*1');
+
+    await clickButton('Next', '/authenticate');
+
+    await verifyUserInfo(page, expect);
+
+    await cdpSession.send('WebAuthn.removeVirtualAuthenticator', {
+      authenticatorId: authenticator.authenticatorId,
+    });
```

Comment:
It seems like we're just sending the WebAuthn credentials and then stopping the test. Am I missing something?

Reply:
after verifying the user info i remove the authenticator

Reply:
In addition, just sent a new commit with failure and successful cases

Reply:
@juangarmendia05 maybe can you record a test and upload what it looks like on the PR here?